pkg/gce-pd-csi-driver/controller.go

```diff
@@ -485,12 +486,19 @@ func (gceCS *GCEControllerServer) createMultiZoneDisk(ctx context.Context, req *
 	}
 	defer gceCS.volumeLocks.Release(volumeID)
 
+	// If creating an empty disk (content source nil), always create RWO disks (when supported)
+	// This allows disks to be created as underlying RWO disks, so they can be hydrated.
```

Comment:
This means in volume populator we need to update the disk accessMode?

Reply:
No, accessMode is updated when the disk is attached (in controller publish), based on the accessMode of the PV that is using it: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/193aa66666ffba45fee3e3d60ac40e7b9453987f/pkg/gce-pd-csi-driver/controller.go#L1167

So this allows for:
1) An empty RWX PV/PVC to be created with underlying disks in RWO mode
2) A static RWO PV/PVC to be created, with the volumeHandle from (1) to hydrate the disk in RWO mode
3) A consumer to use the RWX PV/PVC from (1) after data has been hydrated